music_assistant/server/providers/apple_music/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "music",
+  "domain": "apple_music",
+  "name": "Apple Music",
+  "description": "Support for the Apple Music streaming provider in Music Assistant.",
+  "codeowners": ["@MarvinSchenkel"],
+  "requirements": ["m3u8==4.1.0", "pywidevine==1.8.0"],
```

Comment:
is this m3u8 dependency really needed ? Is there anything in there we cant already handle with our playlists helper ?